challenge_6/python/ajschrier/IntRange.py

```diff
@@ -0,0 +1,41 @@
+ARROW = '->'
+
+
+def outputRange(inputList):
+    # Helper Vars
+    outputList = []
+    rangeStart = None
+    rangeEnd = None
+
+    for i in inputList:
+        # Detect the beginning of the list
+        if not rangeStart:
```

Comment:
You will want to use `if rangeStart is None:`, as a zero value starting the list would be overridden with the second iteration.